backend/src/test/resources/db/migration/V100__populate_test_data.sql

```diff
@@ -0,0 +1,10 @@
+-- data for unit tests
```

Comment:
`V100` - ve jmenu migrace?

Reply:
jo, protoze to je v testech - `backend/src/test/resources` - tzn tam potrebuju buffer kdybychom pridavali nove migrace v realnem kodu, tak aby se nam testy zbytecne nerozbijely

Reply:
A neni tu migraci tim padem lepsi udelat v base testu v kodu? Tj. zbavis se toho cislovani.

Reply:
no tot otazka, vim kam miris, ale prijde mi, ze to je operace navic, protoze ty migrace poleti tak jako tak, proto bych je nechal v SQLku a je tam metoda, kterou kdyz overridujes, tak te necha inicializovat data do databaze

cislovani po stovkach pouzivam v jenom projektu (kdy mam knihovnu ktera obsahuje server a nad ni stavim dalsi data) a funguje to hezky, takze bych to tak asi nechal